CHANGES.rst

```diff
@@ -10,6 +10,12 @@
 Changes
 =======
 
+Version 1.0.1 (Release 2019-08-02)
+----------------------------------
+
+- Remove html sanitization config
```

Comment:
Could you say ``Removes`` instead of ``Remove`` so we are consistent with the documentation? https://github.com/inveniosoftware/invenio/pull/3955/files